### DISCUSSION TITLE
Support windows worker node ?

### DISCUSSION THREAD

**Discussion by jackchuong** [k3s-io/k3s#9099] [Q&A] [answered]
Hi all,
I have setup internal k8s cluster using k3s as below
```
# single master node Ubuntu 22.04
curl -sfL https://get.k3s.io | sh -s - --disable=servicelb,traefik,local-storage
# 2 worker nodes Ubuntu 22.04
curl -sfL https://get.k3s.io | sh -s - agent --token token --server https://<master-node>:6443
```
They work fine.
Now I want to add windows worker node to k8s cluster , is it feasible ?
If you have experience with this, please give me advice , thank you very much.

**Comment by dereknola** [comment, marked as answer]
Windows support for K3s is still very experimental and under development. See https://github.com/k3s-io/k3s/issues/7258
